bank_statement_journal_items/readme/USAGE.rst

```diff
@@ -0,0 +1,5 @@
+To use this module, you need to:
+
+1. Create Bank Statement.
+2. Click on Journal Items button in a bank statement.
+3. Click on Journal item from the list to see detailed information.
```

Comment:
I don't think usage makes sense if the description is spot on, but you could add yourself in `CONTRIBUTORS.rst` :wink: